packages/xo-server-sdn-controller/src/index.js

```diff
@@ -186,15 +186,13 @@ class SDNController extends EventEmitter {
         forOwn(networks, async network => {
           if (network.other_config.private_pool_wide === 'true') {
             log.debug(
-              `Adding network: '${network.name_label}' for pool: '${
-                network.$pool.name_label
-              }' to managed networks`
+              `Adding network: '${network.name_label}' for pool: '${network.$pool.name_label}' to managed networks`
```

Comment:
Do you know why the formatting changed?

Reply:
No, I was looking into it.
It didn't appear when i did `git add --patch` so this must be the hook.
But why did it changed it now and not before, i have no idea...

Reply:
Texts haven't changed, only indentation.

Reply:
I think this is due to an update of Prettier on `master`.

Make sure you are up-to-date (ie rebase) and that latest deps are installed (`yarn`).